ccf/tests/test_util.py

```diff
@@ -0,0 +1,7 @@
+from django.contrib.auth.models import User
```

Comment:
Please call this file test_utils.py, theoretically there will be many utilities in this file, not just one.

Reply:
*sigh*